no-littering.el

```diff
@@ -234,6 +234,7 @@ This variable has to be set before `no-littering' is loaded.")
        `(make-directory ,(var "elfeed/") t))
     (setq elfeed-db-directory              (var "elfeed/db/"))
     (setq elfeed-enclosure-default-dir     (var "elfeed/enclosures/"))
+    (setq rmh-elfeed-org-files             (list (var "elfeed/elfeed.org")))
```

Comment:
(20) But somehow this comment got moved from 4e22b51f081381dfbc9577a066687338e3cf2af7 to dd162629c6bc3d06bd8dca9cf800a75d58475c08, which is at least surprising.